{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: authentication fail\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/topologies/replset.js:1195:38\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:750:7\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:726:20\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/auth/scram.js:191:18\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:67:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:122:9)","errors":[{"name":"ds149761-a1.mlab.com:49761","err":{"name":"MongoError","message":"Authentication failed.","ok":0,"code":18,"errmsg":"Authentication failed."}},{"name":"ds149761-a0.mlab.com:49761","err":{"name":"MongoError","message":"Authentication failed.","ok":0,"code":18,"errmsg":"Authentication failed."}}],"level":"warn","timestamp":"2017-05-22T00:36:55.405Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: authentication fail\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/topologies/replset.js:1195:38\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:750:7\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:726:20\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/auth/scram.js:191:18\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:67:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:122:9)","errors":[{"name":"ds149761-a1.mlab.com:49761","err":{"name":"MongoError","message":"Authentication failed.","ok":0,"code":18,"errmsg":"Authentication failed."}},{"name":"ds149761-a0.mlab.com:49761","err":{"name":"MongoError","message":"Authentication failed.","ok":0,"code":18,"errmsg":"Authentication failed."}}],"level":"warn","timestamp":"2017-05-22T00:36:55.408Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: authentication fail\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/topologies/replset.js:1195:38\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:750:7\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:726:20\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/auth/scram.js:191:18\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:67:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:122:9)","errors":[{"name":"ds149761-a1.mlab.com:49761","err":{"name":"MongoError","message":"Authentication failed.","ok":0,"code":18,"errmsg":"Authentication failed."}},{"name":"ds149761-a0.mlab.com:49761","err":{"name":"MongoError","message":"Authentication failed.","ok":0,"code":18,"errmsg":"Authentication failed."}}],"level":"warn","timestamp":"2017-05-22T00:36:55.408Z"}
{"error":{"name":"MongoError","message":"authentication fail","errors":[{"name":"ds149761-a0.mlab.com:49761","err":{"name":"MongoError","message":"Authentication failed.","ok":0,"code":18,"errmsg":"Authentication failed."}}]},"level":"error","message":"Error generating response. { MongoError: authentication fail\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/topologies/replset.js:1195:38\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:750:7\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:726:20\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/auth/scram.js:191:18\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:67:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:122:9)\n  name: 'MongoError',\n  message: 'authentication fail',\n  errors: [ { name: 'ds149761-a0.mlab.com:49761', err: [Object] } ] }","timestamp":"2017-05-22T00:37:36.089Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: authentication fail\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/topologies/replset.js:1195:38\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:750:7\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:726:20\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/auth/scram.js:191:18\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:67:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:122:9)\n  name: 'MongoError',\n  message: 'authentication fail',\n  errors: [ { name: 'ds149761-a0.mlab.com:49761', err: [Object] } ] } MongoError: authentication fail\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/topologies/replset.js:1195:38\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:750:7\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:726:20\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/auth/scram.js:191:18\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:461:18\n    at _combinedTickCallback (internal/process/next_tick.js:67:7)\n    at process._tickDomainCallback (internal/process/next_tick.js:122:9)","timestamp":"2017-05-22T00:37:36.092Z"}
{"message":"Unable to ensure uniqueness for usernames: ","name":"MongoError","stack":"MongoError: Authentication failed.\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:489:72\n    at authenticateStragglers (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:435:16)\n    at Connection.messageHandler (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:469:5)\n    at Socket.<anonymous> (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/connection.js:321:22)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at readableAddChunk (_stream_readable.js:176:18)\n    at Socket.Readable.push (_stream_readable.js:134:10)\n    at TCP.onread (net.js:551:20)","ok":0,"code":18,"errmsg":"Authentication failed.","level":"warn","timestamp":"2017-05-22T00:38:36.184Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","name":"MongoError","stack":"MongoError: Authentication failed.\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:489:72\n    at authenticateStragglers (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:435:16)\n    at Connection.messageHandler (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:469:5)\n    at Socket.<anonymous> (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/connection.js:321:22)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at readableAddChunk (_stream_readable.js:176:18)\n    at Socket.Readable.push (_stream_readable.js:134:10)\n    at TCP.onread (net.js:551:20)","ok":0,"code":18,"errmsg":"Authentication failed.","level":"warn","timestamp":"2017-05-22T00:38:36.185Z"}
{"message":"Unable to ensure uniqueness for role name: ","name":"MongoError","stack":"MongoError: Authentication failed.\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:489:72\n    at authenticateStragglers (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:435:16)\n    at Connection.messageHandler (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:469:5)\n    at Socket.<anonymous> (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/connection.js:321:22)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at readableAddChunk (_stream_readable.js:176:18)\n    at Socket.Readable.push (_stream_readable.js:134:10)\n    at TCP.onread (net.js:551:20)","ok":0,"code":18,"errmsg":"Authentication failed.","level":"warn","timestamp":"2017-05-22T00:38:36.186Z"}
{"error":{"name":"MongoError","message":"Authentication failed.","ok":0,"code":18,"errmsg":"Authentication failed."},"level":"error","message":"Error generating response. { MongoError: Authentication failed.\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:489:72\n    at authenticateStragglers (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:435:16)\n    at Connection.messageHandler (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:469:5)\n    at Socket.<anonymous> (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/connection.js:321:22)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at readableAddChunk (_stream_readable.js:176:18)\n    at Socket.Readable.push (_stream_readable.js:134:10)\n    at TCP.onread (net.js:551:20)\n  name: 'MongoError',\n  message: 'Authentication failed.',\n  ok: 0,\n  code: 18,\n  errmsg: 'Authentication failed.' }","timestamp":"2017-05-22T00:39:08.589Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: Authentication failed.\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:489:72\n    at authenticateStragglers (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:435:16)\n    at Connection.messageHandler (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:469:5)\n    at Socket.<anonymous> (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/connection.js:321:22)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at readableAddChunk (_stream_readable.js:176:18)\n    at Socket.Readable.push (_stream_readable.js:134:10)\n    at TCP.onread (net.js:551:20)\n  name: 'MongoError',\n  message: 'Authentication failed.',\n  ok: 0,\n  code: 18,\n  errmsg: 'Authentication failed.' } MongoError: Authentication failed.\n    at Function.MongoError.create (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/error.js:31:11)\n    at /Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:489:72\n    at authenticateStragglers (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:435:16)\n    at Connection.messageHandler (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/pool.js:469:5)\n    at Socket.<anonymous> (/Users/davidokunibm/Afterparty/ServerProd/node_modules/mongodb-core/lib/connection/connection.js:321:22)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at readableAddChunk (_stream_readable.js:176:18)\n    at Socket.Readable.push (_stream_readable.js:134:10)\n    at TCP.onread (net.js:551:20)","timestamp":"2017-05-22T00:39:08.592Z"}
{"className":"Photos","triggerType":"beforeSave","user":"6FEhYpWWL9","level":"info","message":"beforeSave triggered for Photos for user 6FEhYpWWL9:\n  Input: {\"height\":852,\"imageFile\":{\"__type\":\"File\",\"name\":\"40f2730b123d2ef52beeb0c118081b5a_image.jpg\",\"url\":\"https://afterparty-file-storage.s3.amazonaws.com/40f2730b123d2ef52beeb0c118081b5a_image.jpg\"},\"thumbID\":\"THUMBABXce9iCHrzKucCwYA\",\"refID\":\"ABXce9iCHrzKucCwYA\",\"comments\":[],\"width\":439,\"eventID\":\"ABXce9iCHr\",\"reports\":0,\"timestamp\":{\"__type\":\"Date\",\"iso\":\"2017-05-22T00:45:54.809Z\"},\"user\":\"dokun1\",\"thumbFile\":{\"__type\":\"File\",\"name\":\"d557e73cdd2323af8755420fc7a94acc_thumb.jpg\",\"url\":\"https://afterparty-file-storage.s3.amazonaws.com/d557e73cdd2323af8755420fc7a94acc_thumb.jpg\"}}\n  Result: {\"object\":{\"comments\":[],\"timestamp\":{\"__type\":\"Date\",\"iso\":\"2017-05-22T00:45:54.809Z\"},\"height\":852,\"imageFile\":{\"__type\":\"File\",\"name\":\"40f2730b123d2ef52beeb0c118081b5a_image.jpg\",\"url\":\"https://afterparty-file-storage.s3.amazonaws.com/40f2730b123d2ef52beeb0c118081b5a_image.jpg\"},\"thumbID\":\"THUMBABXce9iCHrzKucCwYA\",\"refID\":\"ABXce9iCHrzKucCwYA\",\"width\":439,\"eventID\":\"ABXce9iCHr\",\"reports\":0,\"user\":\"dokun1\",\"thumbFile\":{\"__type\":\"File\",\"name\":\"d557e73cdd2323af8755420fc7a94acc_thumb.jpg\",\"url\":\"https://afterparty-file-storage.s3.amazonaws.com/d557e73cdd2323af8755420fc7a94acc_thumb.jpg\"}}}","timestamp":"2017-05-22T00:46:09.558Z"}
{"className":"Photos","triggerType":"afterSave","user":"6FEhYpWWL9","level":"info","message":"afterSave triggered for Photos for user 6FEhYpWWL9:\n  Input: {\"comments\":[],\"timestamp\":{\"__type\":\"Date\",\"iso\":\"2017-05-22T00:45:54.809Z\"},\"height\":852,\"imageFile\":{\"__type\":\"File\",\"name\":\"40f2730b123d2ef52beeb0c118081b5a_image.jpg\",\"url\":\"https://afterparty-file-storage.s3.amazonaws.com/40f2730b123d2ef52beeb0c118081b5a_image.jpg\"},\"thumbID\":\"THUMBABXce9iCHrzKucCwYA\",\"refID\":\"ABXce9iCHrzKucCwYA\",\"width\":439,\"eventID\":\"ABXce9iCHr\",\"reports\":0,\"user\":\"dokun1\",\"createdAt\":\"2017-05-22T00:46:02.127Z\",\"thumbFile\":{\"__type\":\"File\",\"name\":\"d557e73cdd2323af8755420fc7a94acc_thumb.jpg\",\"url\":\"https://afterparty-file-storage.s3.amazonaws.com/d557e73cdd2323af8755420fc7a94acc_thumb.jpg\"},\"updatedAt\":\"2017-05-22T00:46:02.127Z\",\"objectId\":\"8a8B3RDkcP\"}","timestamp":"2017-05-22T00:46:47.560Z"}
